src/paperless_tesseract/parsers.py

```diff
@@ -8,6 +8,10 @@
 from documents.parsers import ParseError
 from PIL import Image
 
+Image.MAX_IMAGE_PIXELS = os.environ.get(
```

Comment:
Would it make sense to move this to settings.py and reference it here and in parsers.py?

Reply:
I think it would make more sense there.  It's the location where we get basically all variables from the environment